utils/rand/rand.go

```diff
@@ -6,13 +6,18 @@ import (
 	"fmt"
 )
 
-// This package is a wrppaer around true RNG crypto/rand.
-// It implements useful tools using the true RNG and that
-// are not exported by the crypto/rand package.
-// This package does not implement any determinstic RNG (Pseudo RNG)
-// unlike the package flow-go/crypto/random.
+// This package is a wrapper around `crypto/rand` that uses the system RNG underneath
```

Comment:
If you move this comment block up to above `package rand`, and prefix it `// Package rand ...` then Godoc will pick it up as package-level documentation. 

Here's an [example in our codebase](https://github.com/onflow/flow-go/blob/2dd0f32454837a02b0e407cd468f83a2859313ee/cmd/build/version.go#L1-L7) and the [corresponding Godoc output](https://pkg.go.dev/github.com/onflow/flow-go@v0.30.6/cmd/build).